code/modules/mob/living/carbon/xenomorph/Castes/Defiler.dm

```diff
@@ -0,0 +1,198 @@
+/datum/xeno_caste/defiler
+	caste_name = "Defiler"
+	display_name = "Defiler"
+	upgrade_name = "Young"
+	caste_desc = "A frightening looking, bulky xeno that drips with suspect green fluids."

```

Comment:
```suggestion
	caste_desc = "A frightening looking, bulky xeno that drips with suspicious green fluids."
```

Reply:
No, I like suspect more:

1.
not to be relied on or trusted; possibly dangerous or false.

Reply:
suspect is correct here